src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsEmailPreferences/SettingsEditSettingsEmailPreferences.tsx

```diff
@@ -26,16 +26,22 @@ export const SettingsEditSettingsEmailPreferences: React.FC<SettingEditSettingsE
   const emailFrequency = me?.emailFrequency || fallbackFrequency
   const [updated, setUpdated] = useState(false)
 
+  const { sendToast } = useToasts()
+
   const handleSelect = async newEmailFrequency => {
     setUpdated(false)
     const variables = { emailFrequency: newEmailFrequency }
-    await UpdateUserEmailPreferencesMutation(
-      // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-      relayEnvironment,
-      variables,
-      me?.id
-    )
-    setUpdated(true)
+    try{
+      await UpdateUserEmailPreferencesMutation(
+        // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
+        relayEnvironment,
+        variables,
+        me?.id
+      )
+      setUpdated(true)
```

Comment:
You'd want to send a success toast here "Preferences updated" and get rid of the updated flag and banner below

Reply:
Updated.